locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/BrooklynImageChooser.java

```diff
@@ -73,7 +73,7 @@ protected static boolean imageNameContainsWordCaseInsensitive(Image img, String
         return img.getName().toLowerCase().matches("(.*[^a-z])?"+pattern.toLowerCase()+"([^a-z].*)?");
     }
     
-    public double punishmentForOldOsVersions(Image img, OsFamily family, double minVersion) {
+    private double punishmentForOldOsVersions(Image img, OsFamily family, double minVersion) {
```

Comment:
Can you make this `protected`. If someone needs to fine-tune the selection the easiest thing to do would be to extend this class and configure an override in the location.